openfe/protocols/openmm_afe/base.py

```diff
@@ -189,8 +191,12 @@ def _pre_equilibrate(
           Equilibrated system positions
         """
         # Prep the simulation object
-        platform = compute.get_openmm_platform(
-            settings['engine_settings'].compute_platform
+        # Restrict CPU count if running vacuum simulation
+        restrict_cpu = settings['forcefield_settings'].nonbonded_method.lower() == 'nocutoff'
```

Comment:
another argument we really should have an explicit way of saying this is a vacuum simulation. I propose we add a setting somewhere for that https://github.com/OpenFreeEnergy/openfe/issues/904 

In the meantime, this seems like a pretty good heuristic.  

We could do more logging, I it would be nice to do a hackathon on it but in the mean time I will just suggest as I see it. It would be good to log what is going on here, maybe could be more verbose than what I suggest but this seems like a spot where if someone was like "why is this running on the CPU and not the GPU?" a log message could hep

```suggestion
        restrict_cpu = settings['forcefield_settings'].nonbonded_method.lower() == 'nocutoff'
        logging.info(f"{restrict_cpu=}")
```